sorc/chgres_cube.fd/wam_climo_data.f90

```diff
@@ -0,0 +1,2758 @@
+!!       for the most physically realistic calculation these three      
+!!       variables should be consistent (stl=sec/3600+glong/15).        
+!!       the equation of time departures from the above formula         
+!!       for apparent local time can be included if available but       
+!!       are of minor importance.                                       
+!!                                                                      
+!!       f107 and f107a values used to generate the model correspond    
+!!       to the 10.7 cm radio flux at the actual distance of the earth  
+!!       from the sun rather than the radio flux at 1 au.               
+!!                                                                      
+!! output variables:                                                 
+!! @param [out] d density array with length of 9
+!! @param [out] t temperature array with length of 2
+!!              d array contains:
+!!               (1) - he number density(cm-3)                                 
+!!               (2) - o number density(cm-3)                                  
+!!               (3) - n2 number density(cm-3)                                 
+!!               (4) - o2 number density(cm-3)                                 
+!!               (5) - ar number density(cm-3)                                 
+!!               (6) - total mass density(gm/cm3) [includes anomalous oxygen]  
+!!               (7) - h number density(cm-3)                                  
+!!               (8) - n number density(cm-3)                                  
+!!               (9) - anomalous oxygen number density(cm-3)                   
+!!              t array conyains:
+!!               (1) - exospheric temperature                                  
+!!               (2) - temperature at alt                                      
+!!                                                                      
+!! @author Hann-Ming Henry Juang
+      subroutine gtd7d(iyd,sec,alt,glat,glong,stl,f107a,f107,ap,mass,d,t)
+      use wam_gtd7bk_mod, only: imr
+!x----common/metsel/imr 
+      dimension d(9),t(2),ap(7),ds(9),ts(2) 
+      call gtd7(iyd,sec,alt,glat,glong,stl,f107a,f107,ap,mass,d,t) 
+!       total mass density                                              
+!                                                                       
+        if(mass.eq.48) then 
+         d(6) = 1.66e-24*(4.*d(1)+16.*d(2)+28.*d(3)+32.*d(4)+40.*d(5)+  &
+             d(7)+14.*d(8)+16.*d(9))                                    
+         if(imr.eq.1) d(6)=d(6)/1000. 
+         endif 
+      return 
+      end subroutine gtd7d
+
+!-----------------------------------------------------------------------
+!> Find altitude of pressure surface (press) from gtd7             
+!!
+!!    input:                                                            
+!! @param [in] iyd - year and day as yyddd                                    
```

Comment:
No space between param and [in]. Also don't use a hyphen to separate the description from the var name. Doxygen already does that and the hypen does not help. So do this instead:

`@param[in] iyd year and day as yyddd  `

Reply:
ok